### PR TITLE
executor: don't output foreign keys in show create table statement.

### PR DESCRIPTION
**Pull request by winkyao**
Ignore foreign keys in `show create table` statement. To fix the problem when restoring data to MySQL.

### CODE REVIEW

File: executor/show.go [hunk 1]
```diff
@@ -453,6 +453,47 @@ func getDefaultCollate(charsetName string) string {
 	return ""
 }
 
+func (e *ShowExec) showForeignKeys(buf *bytes.Buffer, tb table.Table) {
```
Comment (shenli) [inline executor/show.go]: We could just remove fk related code.

File: executor/show.go [hunk 2]
```diff
@@ -527,10 +527,6 @@ func (e *ShowExec) fetchShowCreateTable() error {
 		buf.WriteString(fmt.Sprintf("  PRIMARY KEY (`%s`)", pkCol.Name.O))
 	}
 
-	if len(tb.Indices()) > 0 || len(tb.Meta().ForeignKeys) > 0 {
```
Comment (shenli) [inline executor/show.go]: if len(tb.Indices()) > 0 , we need a `\n`?

Comment (winkyao) [reply executor/show.go]: Done

**Comment by shenli** [comment]
LGTM

**Pull request merged**